WordPress/src/test/java/org/wordpress/android/ui/stats/refresh/lists/sections/insights/usecases/FollowersUseCaseTest.kt

```diff
@@ -351,7 +350,7 @@ class FollowersUseCaseTest : BaseUnitTest() {
     private fun List<BlockListItem>.assertViewAllFollowersSecondLoad() {
         assertThat(this).hasSize(14)
 
-        val follower = this[13] as ListItemWithIcon
+        val follower = this[12] as ListItemWithIcon
```

Comment:
I wanted to confirm if this is the expected change (from `this[13]` to `this[12]`)

Reply:
👋 @RenanLukas !

[SAME](https://github.com/wordpress-mobile/WordPress-Android/pull/17673#discussion_r1053123486), plus notice the fact that I added the `advanceUntilIdle()` and replaced the `Dispatchers.Unconfined` with the `testDispatcher()` during setup, that is, in addition to replacing the `TEST_DISPATCHER` with the `testDispatcher()`. I am actually not sure why it was done like that on all those use cases, but I thought this is a good opportunity to get rid of this customization for good. Thus, I am almost certain that this is why this assertion had to be updated as well.